pkg/webhook/policy.go

```diff
@@ -193,6 +193,22 @@ func (h *validationHandler) Handle(ctx context.Context, req admission.Request) a
 		return admission.Errored(http.StatusInternalServerError, err)
 	}
 
+	if *logStatsAdmission {
+		logging.LogStatsEntries(
+			h.opa,
+			log.WithValues(
+				logging.Process, "admission",
+				logging.EventType, "review_response_stats",
+				logging.ResourceGroup, req.AdmissionRequest.Kind.Group,
+				logging.ResourceAPIVersion, req.AdmissionRequest.Kind.Version,
+				logging.ResourceKind, req.AdmissionRequest.Kind.Kind,
+				logging.ResourceNamespace, req.AdmissionRequest.Namespace,
+				logging.RequestUsername, req.AdmissionRequest.UserInfo.Username,
+			),
+			resp.StatsEntries, "admission review request stats",
```

Comment:
We should verify that the way expansion aggregation is currently coded doesn't interfere with resp.StatsEntries (see source code for reviewRequest(), `expansion.AggregateResponses(res.TemplateName, resp, resultantResp)`)

Reply:
Yep, we should include them. It'll be a good test case for whether it's possible to intelligently aggregate stats responses.

Reply:
To clarify, aggregation of expanded resources' stats was included in the https://github.com/open-policy-agent/gatekeeper/pull/2686/commits/45d80d85b3581eb3073918311be7503ae5b57a4b commit